api/src/paths/dwc/submission/{datasetId}/artifacts.ts

```diff
@@ -134,12 +152,27 @@ export function getArtifactsByDatasetId(): RequestHandler {
       const artifacts = await artifactService.getArtifactsByDatasetId(datasetId);
 
       const artifactsWithRules = await Promise.all(
-        artifacts.map(async (artifact) => {
-          const appliedSecurity = await securityService.getSecurityAppliedStatus(artifact.artifact_id);
-          return {
-            ...artifact,
-            supplementaryData: { persecutionAndHarm: appliedSecurity }
-          };
+        artifacts.map((artifact) => {
+          return securityService
+            .getPersecutionAndHarmRulesByArtifactId(artifact.artifact_id)
+            .then((persecutionAndHarmRules) => {
+              let persecutionAndHarmStatus: SECURITY_APPLIED_STATUS = SECURITY_APPLIED_STATUS.PENDING;
+
+              if (artifact.security_review_timestamp) {
+                persecutionAndHarmStatus =
+                  persecutionAndHarmRules.length > 0
+                    ? SECURITY_APPLIED_STATUS.SECURED
+                    : SECURITY_APPLIED_STATUS.UNSECURED;
+              }
```

Comment:
Regarding `persecutionAndHarmStatus`, this comment is probably relevant here as well for the same reasons: https://github.com/bcgov/biohubbc-platform/pull/205/files#r1246871748

Reply:
If this PR gets merged, I was going to handle on SIMSBIOHUB-10